packages/expo-maps/src/index.ts

```diff
@@ -36,4 +36,18 @@ export namespace AppleMaps {
   export type Marker = AppleTypes.Marker;
 }
 
```

Comment:
Would be good to provide a hook for permissions like we have done in other packages
https://github.com/expo/expo/blob/4520d963727267acfbce2d1786ddf0afcaec253a/packages/expo-camera/src/index.ts#L37C1-L40C4

Reply:
Added here https://github.com/expo/expo/pull/34643/commits/509101bf4b5cf77ce26b12b3ab3e4766edb78949#diff-7517ff5a3a5b8c3f5089a3964f47ef24911cde0a8b791bf35179a07377c19a50R41-R53